assets/js/theme/cart.js

```diff
@@ -139,8 +139,8 @@ export default class Cart extends PageManager {
             this.bindGiftWrappingForm();
         });
 
-        utils.hooks.on('product-option-change', (event, option) => {
-            const $changedOption = $(option);
+        utils.hooks.on('product-option-change', (event, currentTarget) => {
```

Comment:
Don't we need to change places where we do the `bind` to pass `currentTarget` for all of them ?

Reply:
I've already changed it in stencil-utils:
https://github.com/bigcommerce/stencil-utils/blob/master/src/hooks/product.js